src/main/java/hello/Application.java

```diff
@@ -0,0 +1,55 @@
+package hello;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class Application {
+
+    private static final Logger log = LoggerFactory.getLogger(Application.class);
+
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class);
+    }
+
+    @Bean
+    public CommandLineRunner demo(CustomerRepository repository) {
```

Comment:
the purpose of this code to me it seems that you are populating the db and doing some operations on it.  Please move this into your test class.  Find out how to "setup" your test cases and create appropriate tests for the operations that are done here.